lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -115,38 +117,28 @@ foam.CLASS({
   ],
 
   methods: [
-    function init() {
-      this.validate();
-      this.ensureFetched_().then(this.onFetched);
-    },
-    function put(obj) {
-      return Promise.reject('LocalGitIDLFileDAO: DAO is read-only.');
-    },
-    function remove(obj) {
-      return Promise.reject('LocalGitIDLFileDAO: DAO is read-only.');
-    },
-    function find(objOrId) {
-      return this.checkNotReady_() || this.delegate.find(objOrId);
-    },
-    function select(sink, skip, limit, order, predicate) {
-      return this.checkNotReady_() ||
-          this.delegate.select(sink, skip, limit, order, predicate);
-    },
-    function removeAll(skip, limit, order, predicate) {
-      return Promise.reject('LocalGitIDLFileDAO: DAO is read-only.');
-    },
+    function run(args) {
```

Comment:
Why is this receiving arbitrary args? We should either:

(1) Pass initialization data on `.create({ <init data> })`, ignore arg in `run(arg)`, and ensure that `factory` and `expression` are used appropriately to wire up dependent config data,

or

(2) Model a config object and can be checked with `iInstance()` and/or `.validate()` before being trusted inside `run()`.

I prefer (1) if all config data is known when we go to create instances of this class.